src/integration-test/java/org/ow2/proactive/catalog/rest/controller/AbstractCatalogObjectRevisionControllerTest.java

```diff
@@ -43,28 +43,28 @@
  */
 public class AbstractCatalogObjectRevisionControllerTest extends AbstractRestAssuredTest {
 
-    protected static final String CATALOG_OBJECT_RESOURCE = "/buckets/{bucketId}/resources/{objectId}";
+    protected static final String CATALOG_OBJECT_RESOURCE = "/buckets/{bucketId}/resources/{name}";
 
     protected static final String CATALOG_OBJECTS_RESOURCE = "/buckets/{bucketId}/resources/";
 
-    protected static final String CATALOG_OBJECT_REVISIONS_RESOURCE = "/buckets/{bucketId}/resources/{objectId}/revisions";
+    protected static final String CATALOG_OBJECT_REVISIONS_RESOURCE = "/buckets/{bucketId}/resources/{name}/revisions";
 
-    protected static final String CATALOG_OBJECT_REVISION_RESOURCE = "/buckets/{bucketId}/resources/{objectId}/revisions/{commitId}";
+    protected static final String CATALOG_OBJECT_REVISION_RESOURCE = "/buckets/{bucketId}/resources/{name}/revisions/{commitTime}";
```

Comment:
yes, name must be unique in the bucket, and the commit time is unique in the object in the bucket. i leave the id of the revision because spring data need an id, but it could be refactored again to have bucketid, name and commitTime as a composite key.